README.md

```diff
@@ -137,19 +142,23 @@ WARNING: All illegal access operations will be denied in a future release
 ```
 
 This warning, while valid, produces noise in an organization's logging infrastructure. In Java 17, these types of issues
-will are now fatal and the application will terminate if such illegal access occurs. 
+are now fatal and the application will terminate if such illegal access occurs.
 
 ### Suppressing Illegal Reflective Access Exceptions.
 
-In situations where there a third-party library does not provide a version that is compliant with the Java module system,
-it is possible to suppress these warnings/errors. An application may add `Add-Opens` declarations to it's top-level JAR's manifest:
+In situations where there a third-party library does not provide a version that is compliant with the Java module
+system, it is possible to suppress these warnings/errors. An application may add `Add-Opens` declarations to its
+top-level JAR's manifest:
 
 ```xml
-<Add-Opens>java.base/java.lang java.base/java.util java.base/java.lang.reflect java.base/java.text java.desktop/java.awt.font</Add-Opens>
+
+<Add-Opens>java.base/java.lang java.base/java.util java.base/java.lang.reflect java.base/java.text
+    java.desktop/java.awt.font
+</Add-Opens>
```

Comment:
This one rendered differently; perhaps this would be slightly preferred then?
```suggestion
<Add-Opens>
    java.base/java.lang java.base/java.util java.base/java.lang.reflect java.base/java.text java.desktop/java.awt.font
</Add-Opens>
```